### DISCUSSION TITLE
How to install php-amqp.so in synology

### DISCUSSION THREAD

**Discussion by xorgxx** [rabbitmq/rabbitmq-server#4966] [Other] [answered]
Hi i using RabbitMq MQTT for many projet. it live on my owne server DS216 synology in docker container.
so far all work perfect with Symfony project. 

i want to push more deep !! i want to use not only with MQTT but extension of the Symfony Messenger, so for this to be in order i have to install php-amqp.so as extension module to PHP8 server.

i have read a lot of tuto .. bla bla bla bla but none of them explaine exatly how i can do this. 
i dont have any apt-get or staff lake this to intall.

for info i have juste install redis.so and it work find. 

thank for any help to have this extension in PHP8

**Comment by michaelklishin** [comment, marked as answer]
This is a question for the PHP community: the MQTT client library and whatever Symfony extension you plan on using.

And leave out the “blah blah” part, it makes you sounds incredibly disrespectful. The authors of open source software, docs and tutorials do not owe you anything.
